dns/zone-configs/k8s.io._0_base.yaml

```diff
@@ -19,6 +19,9 @@
   - type: TXT
     values:
     - google-site-verification=RJbZ_ganmSWvslSKOBG-QHv62XTjJZcigpWIFttStFs
+    # Domain validation for k8s.io on https://search.google.com/search-console/welcome
+    # Only Org admins have access to the console. (contact @ameukam)
+    - google-site-verification=dgC0yQp0oE3cj8yqkbQTfPmStJmi1Qaha_MHwM4Sa10
```

Comment:
So what's the `google-site-verification` above then?  I would like to know who has registered as owner of the domain via that (@thockin? @ixdy?), and whether this overrides that

Reply:
I don't think I have access to this.

Reply:
Might be for the website? (not sure - that level of infrastructure has mostly “just worked” so I haven't looked into it and I also don't know where it might be documented)

Reply:
When I go to the console I see both URLs.

Property added to account
October 31, 2014

So where is this new verification coming from?

Reply:
Apparently I am not a verified owner of kubernetes.io unless we verify, but k8s.io already has the verification HTML file uploaded

Reply:
I have no clue how to share ownership... I can not find it..